packages/generator/templates/app/blitz.config.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   middleware: [
     sessionMiddleware({
       isAuthorized: simpleRolesIsAuthorized,
+      cookiePrefix: '__name__',
```

Comment:
A bit pedantic but I think this reads better :)
```suggestion
      cookiePrefix: '__name__',
      isAuthorized: simpleRolesIsAuthorized,
```